riscv-overlay-software-standard-draft.adoc

```diff
@@ -70,25 +70,25 @@ link:#instrumentation-interface-hooks[Table 1 - Instrumentation interface hooks]
 [[overview]]
 == Overview
 
-Physical memory is one of the most important resource in sw programing, since the program runs on it. Some embedded systems have limited memory resources and as a result the total code footprint is bigger than the available memory. To solve this problem, the program, needs to use a technique to load executable code on run time from virtual memory or to overrun executable code which is not needed anymore by the program.
+Physical memory is one of the most important resource in software programing, since the program runs on it. Some embedded systems have limited memory resources and as a result the total code footprint is bigger than the available memory. To solve this problem, the program, needs to use a technique to load executable code on run time from virtual memory or to overrun executable code which is not needed anymore by the program.
```

Comment:
if we have SW on the abbreviation table why we need to write ```software``` ?

Reply:
We don't need to. I just thought it read smoother in the beginning of introduction to use the whole word, but I have no strong preference.

Reply:
ok